### PR TITLE
#20272 Make keep alive setting clearer

### LINKED ISSUE #20272: Make Keep-Alive setting clearer (improve name and tooltip)
### Description

In DBeaver 23.1.0 seeing behavior (with at least the Vertica driver) where `select current_schema()` is being triggered every second.

```
dbadmin=> select request, count(*), max(start_timestamp) from query_requests where user_name ='alattke' group by 1 order by 2 desc limit 1;
         request         | count |              max              
-------------------------+-------+-------------------------------
 select current_schema() |  1100 | 2023-06-12 15:27:48.392504-04
(1 row)

dbadmin=> select request, count(*), max(start_timestamp) from query_requests where user_name ='alattke' group by 1 order by 2 desc limit 1;
         request         | count |              max              
-------------------------+-------+-------------------------------
 select current_schema() |  1268 | 2023-06-12 15:32:00.563201-04
(1 row)
```

I don't see a way to disable this in the settings.

### DBeaver Version

23.1.0 Community Edition

### Operating System

Both Mac and Windows

### Database and driver

Vertica 12.x

### Steps to reproduce

Connect to Vertica. Don't do anything for a few mins. Check the Vertica `query_requests` table.
```
dbadmin=> select request, count(*), max(start_timestamp) from query_requests where user_name ='alattke' group by 1 order by 2 desc limit 1;
         request         | count |              max              
-------------------------+-------+-------------------------------
 select current_schema() |  1100 | 2023-06-12 15:27:48.392504-04
(1 row)

dbadmin=> select request, count(*), max(start_timestamp) from query_requests where user_name ='alattke' group by 1 order by 2 desc limit 1;
         request         | count |              max              
-------------------------+-------+-------------------------------
 select current_schema() |  1268 | 2023-06-12 15:32:00.563201-04
(1 row)
```

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by E1izabeth**
image

image

The link to the docs will be working after the new article publishing soon.

### CODE REVIEW

**Review by ShadelessFox — COMMENTED** [review]
I am concerned about the link in the tooltip. You can't click on it, but you also can't write it down normally because tooltips close after a short delay.

Have you considered a link below controls in this group?

**Comment by LonwoLonwo** [comment]
I 100% agree with Yan this time. This is the reason why I didn't approve it yet.

**Comment by E1izabeth** [comment]
I didn't think about it, just implement exactly what we decided earlier. It's a good idea to add a link to the wiki below. Thanks

File: plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/dialogs/connection/ConnectionPageInitialization.java [hunk 1]
```diff
@@ -60,6 +58,8 @@ class ConnectionPageInitialization extends ConnectionWizardPage implements IData
 
     private static final Log log = Log.getLog(ConnectionPageInitialization.class);
 
+    private static final String PAGE_DOCS_LINK = "https://dbeaver.com/docs/dbeaver/Configure-Connection-Initialization-Settings/";
```
Comment (LonwoLonwo) [inline plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/dialogs/connection/ConnectionPageInitialization.java:61]: But our wiki doesn't have this page. And keep-alive info...

Comment (E1izabeth) [reply plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/dialogs/connection/ConnectionPageInitialization.java:61]: It will be published soon

Comment (E1izabeth) [reply plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/dialogs/connection/ConnectionPageInitialization.java:61]: https://github.com/dbeaver/tech-docs/issues/200

File: plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/dialogs/connection/ConnectionPageInitialization.java [hunk 2]
```diff
@@ -300,7 +300,17 @@ public void widgetSelected(SelectionEvent e) {
         gd.grabExcessHorizontalSpace = true;
         infoLabel.setLayoutData(gd);
         infoLabel.setToolTipText(CoreMessages.dialog_connection_wizard_connection_init_hint_tip);
-
+        Link urlHelpLabel = UIUtils.createLink(
+            group,
+            "<a href=\"" + HelpUtils.getHelpExternalReference(PAGE_DOCS_LINK) + "\">"
```
Comment (ShadelessFox) [inline plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/dialogs/connection/ConnectionPageInitialization.java]: You can use `href="#"` here

File: plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/dialogs/connection/ConnectionPageInitialization.java (hunk 1 above)
Comment (ShadelessFox) [reply plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/dialogs/connection/ConnectionPageInitialization.java:61]: Yup, this link points to nowhere. This is a no-no.

**Review by ShadelessFox — DISMISSED** [review]

**Review by ShadelessFox — APPROVED** [review]

**Review by LonwoLonwo — APPROVED** [review]

**Comment by uslss** [comment]
link to the documentation is [incorrect ](https://dbeaver.com/docs/wiki/https:/dbeaver.com/docs/dbeaver/Configure-Connection-Initialization-Settings/)

**Comment by E1izabeth** [comment]
The documentation is not published yet. I wrote it in the description to this PR.

**Comment by uslss** [comment]
verified

**Pull request merged**